workbench-core/swb-common-ui/src/pages/README.md

```diff
@@ -0,0 +1 @@
+This folder is intentionally kept empty for compiling as a React UI package
```

Comment:
Why do we need the `pages` folder in `swb-common-ui`?

Reply:
Been trying a lot of permutations to get React build working on this package, and one of those attempts required creating the `/pages` folder. Removed it now, seems like currently it's not required.

Reply:
I believe it's working because of this switching off this rule : https://github.com/awslabs/solution-spark-on-aws/blob/fix-chore-separate-UI-common-packages/workbench-core/swb-common-ui/.eslintrc.js#L12

Reply:
Oh this package isn't next specific. It's just common ui components that any ui code base can import and use.